src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./HomePageParagraph.module.css";
+
+const HomePageParagraphBlock = ({ text }) => <p className={s.text}>{text}</p>;
```

Comment:
Since this is so short and only used in one place, I think it'd be cleaner to just inline this into the main component rather than defining another component for it.